tests/python/gym/test_gym_interface.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 pytest.importorskip("gym")
+pytest.importorskip("gym.envs.atari")
```

Comment:
There's no way to hack PYTHONPATH where we could perform Gym tests without having to install `ale-py`. Skip these tests if we don't have `gym.envs.atari` installed.